arkane/mainTest.py

```diff
@@ -35,29 +35,31 @@
 import unittest
 import os
 import shutil
+from nose.plugins.attrib import attr
 
 import rmgpy
 from arkane import Arkane
 
 ################################################################################
 
 
+@attr('functional')
 class TestArkaneExamples(unittest.TestCase):
     """
     Run all of Arkane's examples, and report which one failed
     """
     @classmethod
-    def setUp(self):
+    def setUpClass(cls):
         """A function that is run ONCE before all unit tests in this class."""
-        self.base_path = os.path.join(os.path.dirname(os.path.dirname(rmgpy.__file__)), 'examples', 'arkane')
-        self.failed = []
-        self.example_types = ['species', 'reactions', 'networks']
+        cls.base_path = os.path.join(os.path.dirname(os.path.dirname(rmgpy.__file__)), 'examples', 'arkane')
+        cls.failed = []
+        cls.example_types = ['species', 'reactions', 'networks']
 
     def test_arkane_examples(self):
         for example_type in self.example_types:
             example_type_path = os.path.join(self.base_path, example_type)
             examples = []
-            for (dirpath, dirnames, filenames) in os.walk(example_type_path):
+            for (dirpath, dirnames, filenames) in os.listdir(example_type_path):
```

Comment:
Sorry, I wasn't clear about using `os.listdir`. You should be able to replace
```
examples = []
for (dirpath, dirnames, filenames) in os.walk(example_type_path):
    examples.extend(dirnames)
    break
for example in examples:
    ...
```
with
```
for example in os.listdir(example_type_path):
    ...
```
similar to what you did in the `tearDownClass` method.